src/com_tjvendors/admin_language/en-GB/en-GB.com_tjvendors.ini

```diff
@@ -147,7 +147,7 @@ COM_TJVENDORS_VENDORFEE_DUPLICATE_CURRENCY="Currency Already Created. You can up
 
 ;payouts
 COM_TJVENDORS_FORM_DESC_PAYOUT_TOTAL_PAYABLE_AMOUNT="Enter the amount you want to pay"
-COM_TJVENDOR_PAYOUTS_SEARCH_BY_VENDOR_TITLE="Search by Vendor Title, Currency"
+COM_TJVENDOR_PAYOUTS_SEARCH_BY_VENDOR_TITLE="Search by Currency"
```

Comment:
@deepgawade89 Have you removed the functionality also?

Reply:
Search by vendor title functionality is already not present in the model

Reply:
add it then, as it is quite basic expectation of search to search in all text columns show on list

Reply:
@deepgawade89